### PR TITLE
Suppress exportString tests that fail with CHPL_TARGET_COMPILER=cray-prgenv-cray

### PR DESCRIPTION
**Pull request by dlongnecke-cray**
This PR suppresses two tests that currently fail in a cray-xc environment with CHPL_TARGET_COMPILER=cray-prgenv-cray. The Chapel type `c_ptr(c_char)` currently maps to `int8_t*` on most systems, which causes the GNU compiler to emit warnings that can be suppressed with `-Wno-pointer-sign`. There is no equivalent for Cray CC, so we should suppress these tests until https://github.com/chapel-lang/chapel/issues/6041 is resolved.

### CODE REVIEW

**Comment by dlongnecke-cray** [comment]
@gbtitus, I am unsure of how to replicate the testing environment listed below:
```
    CHPL_HOST_PLATFORM: cray-xc
    CHPL_HOST_COMPILER: gnu
    CHPL_HOST_ARCH: x86_64
    CHPL_TARGET_PLATFORM: cray-xc *
    CHPL_TARGET_COMPILER: cray-prgenv-cray
    CHPL_TARGET_ARCH: x86_64
    CHPL_TARGET_CPU: ivybridge
    CHPL_LOCALE_MODEL: flat
    CHPL_COMM: none *
    CHPL_TASKS: qthreads
    CHPL_LAUNCHER: none *
    CHPL_TIMERS: generic
    CHPL_UNWIND: none
    CHPL_MEM: jemalloc
    CHPL_ATOMICS: intrinsics
    CHPL_GMP: system
    CHPL_HWLOC: hwloc
    CHPL_REGEXP: re2
    CHPL_LLVM: none
    CHPL_AUX_FILESYS: none
    CHPL_LIB_PIC: none
```

When I run the tests in `test/interop/C/exportString` the C tests are skipped. Additionally my $(CC) compiler is clang-9. Is there some module loading/unloading I have to do to match the nightly testing environment, here? (I suspect there is, and will look into it, just a prelude question if you will).

**Comment by gbtitus** [comment]
> @gbtitus, I am unsure of how to replicate the testing environment listed below:

Yes, you'll have to do some module adjustment to replicate this environment.  Generally: on the system where the corresponding testing is done, make sure the PrgEnv module you have loaded is PrgEnv-cray.  That selects the target compiler.  You'll also need to `export CHPL_COMM=none` explicitly, because the default comm layer on XC systems is ugni.  And you'll need to `export CHPL_LAUNCHER=none` explicitly, because the default launcher on XC systems is whatever (PBS or slurm) is installed on the system.

File: test/interop/C/exportString/noArgsRetString.suppressif [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#
+# The Cray CC is a bit more strict about pointer conversions and does not have
+# a flag equivalent to "-Wno-pointer-sign".
```
Comment (gbtitus) [inline test/interop/C/exportString/noArgsRetString.suppressif]: You might adjust the phrasing to say '... a flag equivalent to gcc's "-Wno-pointer-sign"', so it's obvious where someone curious about what that is would go to find out more.  (same for the other file also, of course)

**Comment by dlongnecke-cray** [comment]
Thank you for the help!

**Pull request merged**